spot_wrapper/wrapper.py

```diff
@@ -489,8 +341,8 @@ def __init__(
         logger: logging.Logger,
         start_estop: bool = True,
         estop_timeout: float = 9.0,
-        rates: typing.Optional[typing.Dict] = None,
-        callbacks: typing.Optional[typing.Dict] = None,
+        rates: typing.Dict[str, float] = {},
+        callbacks: typing.Dict[str, typing.Callable] = {},
```

Comment:
```suggestion
        rates: typing.Dict[str, float] = None,
        callbacks: typing.Dict[str, typing.Callable] = None,
```

Should avoid mutable default arguments, see https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument. It probably doesn't matter since the wrapper is only initialised a single time, but better to avoid. This does mean that technically the type hint is incorrect.

Reply:
Agreed, done in [392da89](https://github.com/bdaiinstitute/spot_wrapper/pull/6/commits/392da898d027fc716a17f1cd7ff487183cc192cb)